.travis.yml

```diff
@@ -1,6 +1,13 @@
+sudo: false
 language: node_js
 node_js:
-- '0.10'
+  - 'node'
+  - '4.2'
```

Comment:
this reminds me, we should update this to 4.3 as that's now the latest LTS version.